solidity/complex/defi/UniswapV2Router01/test_evm.json

```diff
@@ -0,0 +1,1044 @@
+{ "cases": [ {
```

Comment:
@gianbelinche we also need target filters here.
But it's not documented, so I'll add it myself!
`"targets": [ "evm" ]`